go/webpack/cbor_format_test.go

```diff
@@ -31,10 +31,14 @@ func TestWriteCBOR(t *testing.T) {
 	pack := Package{
 		parts: []*PackPart{
 			&PackPart{
-				url:            staticUrl("https://example.com/index.html?query"),
-				requestHeaders: HTTPHeaders{},
-				status:         200,
+				requestHeaders: HTTPHeaders{
+					httpHeader(":method", "GET"),
+					httpHeader(":scheme", "https"),
+					httpHeader(":authority", "example.com"),
+					httpHeader(":path", "/index.html?query"),
+				},
 				responseHeaders: HTTPHeaders{
+					httpHeader(":status", "200"),
```

Comment:
Same question about 200 - do we need it as part of the format? If not, it's probably better to remove and add later if/when the need arises.